recipes/gametes/meta.yaml

```diff
@@ -0,0 +1,38 @@
+{% set name = "gametes" %}
+{% set version = "2.1" %}
+
+package:
+  name: {{ name }}
+  version: {{ version }}
+
+about:
+  home: https://sourceforge.net/projects/gametes/
+  license: GNU General Public License version 2.0 (GPLv2)
```

Comment:
Could you use the [SPDX license abbreviation](https://spdx.org/licenses/) here, looks like it should be `GPL-2.0-only`